src/styles/utils/variables.scss

```diff
@@ -0,0 +1,16 @@
+$black: #000;
+$grey: #eee;
+$pink: #ffc0cb;
+
+$day-size: 100px;
+$day-gap: 1px;
+$day-padding: 10px;
+$day-move: $day-size + $day-gap;
+$week:
+  "mon" 0,
+  "tue" $day-move,
+  "wed" $day-move * 2,
+  "thu" $day-move * 3,
+  "fri" $day-move * 4,
+  "sat" $day-move * 5,
+  "sun" $day-move * 6;
```

Comment:
seems like okay, but why you make it confused? you can use just simple numbers instead the expressions with variable ($day-move)